nginx-helper.php

```diff
@@ -9,7 +9,7 @@
  * Text Domain:       nginx-helper
  * Domain Path:       /languages
  * Requires at least: 3.0
- * Tested up to:      6.4
+ * Tested up to:      6.5.2
```

Comment:
We only need to provide the major version, not the minor, so it should be `6.5`

Ref: https://developer.wordpress.org/plugins/wordpress-org/how-your-readme-txt-works/#readme-header-information